cmd/kubeadm/app/preflight/checks.go

```diff
@@ -651,7 +651,9 @@ func (swc SwapCheck) Check() (warnings, errorList []error) {
 	}
 
 	if len(buf) > 1 {
-		return []error{errors.New("swap is supported for cgroup v2 only; the NodeSwap feature gate of the kubelet is beta but disabled by default")}, nil
+		return []error{errors.New("swap is supported for cgroup v2 only; " +
+			"the kubelet must be properly configured to use swap, please refer to https://kubernetes.io/docs/concepts/architecture/nodes/#swap-memory; " +
+			"or disable swap on the node")}, nil
```

Comment:
```suggestion
		return []error{errors.New("swap is supported for cgroup v2 only. " +
			"The kubelet must be properly configured to use swap. Please refer to https://kubernetes.io/docs/concepts/architecture/nodes/#swap-memory, " +
			"or disable swap on the node")}, nil
```

let's switch to proper punctuation.